.vscode/launch.json

```diff
@@ -27,12 +27,20 @@
             "runtimeExecutable": "${execPath}",
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}",
-                "--extensionTestsPath=${workspaceFolder}/out/test"
+                "--extensionTestsPath=${workspaceFolder}/out/tests/suite/index"
```

Comment:
This fixes an existing bug when launching tests using VS Code debugging tools (F5), namely the error:
```
Error: Cannot find module 'c:\code\vscode-aks-tools\out\test'
Require stack:
- c:\Users\pebomber\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\pebomber\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\pebomber\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
```

It's kind of nice to get this fixed because, for reasons I don't understand:
- if I run tests from the command line (`npm run test`) VS Code complains when there's another VS Code process running and refuses to launch the test host.
- if I run tests this way (F5), it happily opens a new host regardless of other VS Code processes running.

Plus, this allows us to debug tests through the IDE, with breakpoints and stepping, etc.